docs/cni-plugin.md

```diff
@@ -35,16 +35,33 @@ Another example with a trunk port and jumbo frames:
 }
 ```
 
+Another example with QinQ configured:
+
+```json
+{
+    "name": "myqinqnet",
+    "type": "ovs",
+    "bridge": "mynet2",
+    "vlan": 1000,
+    "cvlan": [ { "id" : 24 }, { "minID" : 100, "maxID" : 102 } ]
+}
+```
+
 ## Network Configuration Reference
 
 * `name` (string, required): the name of the network.
 * `type` (string, required): "ovs".
 * `bridge` (string, required): name of the bridge to use.
 * `vlan` (integer, optional): VLAN ID of attached port. Trunk port if not
-   specified.
+  specified.
+* `vlanMode` (string, optional): Vlan mode of atached port. One of access, 
+dot1q-tunnel, native-tagged, native-untagged, or trunk. Access if not specified.
 * `mtu` (integer, optional): MTU.
 * `trunk` (optional): List of VLAN ID's and/or ranges of accepted VLAN
   ID's.
+* `cvlan` (optional): List of customer VLAN ID's for 802.1Q tunneling (QinQ).
+ If provided, VLAN ID represents service VLAN. VlanMode needs to be set to dot1q-tunnel.
+ Note that the Open vSwitch needs to have configured vlan-limit=2 in other_config.
```

Comment:
Please add this also above the QinQ config example. Include the example command line to configure that.

Similar for the vlan-limit on a bridge.

Reply:
Added the note to the config example. Included command example.
> Similar for the vlan-limit on a bridge.

AFAIK there is no such config for bridge, is [there](http://www.openvswitch.org//ovs-vswitchd.conf.db.5.pdf)?